### PR TITLE
chore: upgrade toolchain

### PR DESCRIPTION
**Pull request by zhangsoledad**

### CODE REVIEW

**Review by quake — DISMISSED** [review]

File: Makefile [hunk 1]
```diff
@@ -35,7 +35,6 @@ clippy:
 
 ci-crates:
 	@set -eu; \
-	export RUSTFLAGS='-F warnings'; \
```
Comment (yangby-cryptape) [inline Makefile:38]:
```rust
    	export RUSTFLAGS='-D warnings'; \
```

File: Makefile [hunk 2]
```diff
@@ -45,6 +44,5 @@ ci-crates:
 
 ci-example:
 	@set -eu; \
-	export RUSTFLAGS='-F warnings'; \
```
Comment (yangby-cryptape) [inline Makefile:48]:
```rust
    	export RUSTFLAGS='-D warnings'; \
```

File: .travis.yml [hunk 3]
```diff
@@ -25,7 +25,6 @@ matrix:
       rust:
         - stable
       env:
-        - RUSTFLAGS='-F warnings'
```
Comment (yangby-cryptape) [inline .travis.yml:28]:
```yaml
        - RUSTFLAGS='-F warnings'
```

File: .travis.yml [hunk 4]
```diff
@@ -35,9 +34,8 @@ matrix:
       language: rust
       compiler: gcc
       rust:
-        - 1.41.0
+        - 1.51.0
       env:
-        - RUSTFLAGS='-F warnings'
```
Comment (yangby-cryptape) [inline .travis.yml:40]:
```yaml
        - RUSTFLAGS='-F warnings'
```

File: .travis.yml [hunk 5]
```diff
@@ -35,9 +34,8 @@ matrix:
       language: rust
       compiler: gcc
       rust:
-        - 1.41.0
```
Comment (yangby-cryptape) [inline .travis.yml:38]: `1.41.0` is the **minimum** supported rust toolchain version.

**Review by yangby-cryptape — CHANGES_REQUESTED** [review]

**Review by quake — APPROVED** [review]

**Review by yangby-cryptape — APPROVED** [review]

**Pull request merged**